docs/getting-started/getting-started.md

```diff
@@ -43,7 +43,7 @@ $ which git-remote-rad
 <home path>/.radicle/bin/git-remote-rad
 ```
 
-With the remote helper installed, you will be able to [push](using-radicle/pushing-and-pulling-changes.md#pushing-changes-to-a-project) to the Radicle network. 
+With the remote helper installed, you will be able to [push changes](using-radicle/pushing-and-pulling-changes.md#pushing-changes-to-a-project) to the Radicle network.
```

Comment:
```suggestion
With the remote helper installed, you will be able to [push changes](using-radicle/pushing-and-pulling-changes.md#pushing-changes-to-a-project) to your Radicle installation, which will be picked up by the Upstream client.
```

Reply:
What do you mean by Radicle installation?

Reply:
I'm referring to the local radicle monorepo. Pushing itself doesn't result in propagation of the changes to the network, a peer-connected client like Upstream needs to pick it up and communicate.

Reply:
Gotcha